index.html

```diff
@@ -63,7 +63,110 @@ <h4>Skills</h4>
             <a href="#"><img alt= 'Twitter' src="./assets/Twitter.svg"></a>
             <a href="#"><img alt= 'M' src="./assets/M.svg"></a>
         </div>
-        <hr>                    
+        <span>&#8595; &nbsp;&nbsp; WORK</span>
+        <div class="works-flex">
```

Comment:
This section's content should be wrapped in a `section` tag.

<img width="304" alt="Screen Shot 2022-07-07 at 21 38 09" src="https://user-images.githubusercontent.com/82426682/177905828-bb156c76-760a-4cb2-b5fe-13120f5aad9b.png">

Reply:
section tag has been used where required.